lib/components/form/edit.vue

```diff
@@ -0,0 +1,114 @@
+<!--
+Copyright 2017 Super Adventure Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/nafundi/super-adventure/blob/master/NOTICE.
+
+This file is part of Super Adventure. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of Super Adventure,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div id="form-edit" class="panel panel-simple">
+    <div class="panel-heading"><h1 class="panel-title">Form Stage</h1></div>
+    <div class="panel-body">
+      <form>
```

Comment:
need a `<fieldset>` here too to actually group the radiobuttons together. the tabstops are really goofy right now, i think the fieldset will help.

Reply:
relatedly, i think once you put a fieldset in you can simply set disabled on the fieldset and all the radio buttons will disable

Reply:
Very nice, that makes the disabling much more straightforward.

Reply:
I've added the `<fieldset>`, but it didn't seem to have an effect on the tabstops. It seems like the typical behavior with radios it that tabbing brings you to the set of radios, then the arrow keys allow you to move between the radios: https://stackoverflow.com/questions/14322564/can-you-tab-through-all-radio-buttons. However, that's a bit awkward given that we send a request with each radio change...